waterdata/templates/base_plain.html

```diff
@@ -5,6 +5,7 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta name="format-detection" content="telephone=no">
+        {% block description %}{% endblock %}
```

Comment:
Consider using a more general term. This block could be used to any number of meta tags.